marimbabot_audio/README.md

```diff
@@ -0,0 +1,26 @@
+# Sound2Note
```

Comment:
You might want to integrate parts of this README into the main README.

Reply:
done, I have merged this document to the main `README.md`.